### ISSUE TITLE
Automatically log return values of scripts

### ISSUE THREAD

**Issue by alxocity** [onflow/cadence#360]
### Issue To Be Solved

Automatically log return values of scripts so since it would be nice to see the value in the log without having an extra log statement before every return statement.

**Comment by bjartek** [comment]
More loglevels and verbosity options would definately be nice

**Comment by turbolent** [comment]
Agreed, it would be nice to have levels and a way to adjust it in the host environment (Emulator, Playground, etc.), too

**Comment by turbolent** [comment]
@alxocity Were you mainly thinking of the emulator here?

**Comment by bjartek** [comment]
If possible it would be nice to fetch log events associated with a transaction. But depending on the architecture I can see that it could be difficult.

**Comment by turbolent** [comment]
Let's move this to the emulator. Could maybe be something like a `--log-script-results` flag
